Lib/test/test_tarfile.py

```diff
@@ -1899,6 +1899,48 @@ def test_pax_extended_header(self):
+
+        # values that should be changed
+        t = tarfile.TarInfo()
+        t.name = "foo㍴" # can't be represented in ascii
```

Comment:
I prefer to keep the source code as ASCII, it's easier to maintain for developers:

```suggestion
        t.name = "foo\u3374" # can't be represented in ascii
```

Same for the test below.